internal/fileutils/fileutils.go

```diff
@@ -114,11 +131,9 @@ func ReplaceAll(filename, find string, replace string, include includeFunc) erro
 		//logging.Debug("Assuming file '%s' is a text file", filename)
 	}
 
-	replaced := replaceRegex.ReplaceAll(fileBytes, replaceBytes)
-	buffer := bytes.NewBuffer([]byte{})
-	buffer.Write(replaced)
+	replaced := replaceRegex.ReplaceAll(buf, replaceBytes)
 
-	return WriteFile(filename, buffer.Bytes()).ToError()
+	return !bytes.Equal(replaced, buf), replaced, nil
```

Comment:
I did not use `ReplaceAllFunc` here, because the above version of this function already installs an ActivePython project as fast as the most optimized version that I just got rid off.  So, that tells us that optimizing this function has no impact at all.  Therefore, I think that I can avoid the refactoring that `ReplaceAllFunc` would have required.